library/src/G4CMPSolidUtils.cc

```diff
@@ -0,0 +1,245 @@
+/***********************************************************************\
+ * This software is licensed under the terms of the GNU General Public *
+ * License version 3 or later. See G4CMP/LICENSE for the full license. *
+\***********************************************************************/
+
+/// \file library/src/G4PhononReflection.cc
+/// \brief Implementation of the G4PhononReflection class
+//
+// This utility class handles general G4VSolid processes.
```

Comment:
Be more specific here, as I mentioned in the .hh file.  For multiple lines, you need to use the "///" notation for Doxygen to pick up the whole thing.  As currently written, line 9 would not get used by Doxygen.